# This file was autogenerated by uv via the following command:
#    uv pip compile requirements.in --python 3.8 --output-file py38.txt
backports-zoneinfo==0.2.1
    # via -r requirements.in
coverage==7.5.3
    # via -r requirements.in
exceptiongroup==1.2.1
    # via pytest
importlib-metadata==7.1.0
    # via pytest-randomly
iniconfig==2.0.0
    # via pytest
packaging==24.1
    # via pytest
pluggy==1.5.0
    # via pytest
pytest==8.2.2
    # via
    #   -r requirements.in
    #   pytest-randomly
pytest-randomly==3.15.0
    # via -r requirements.in
python-dateutil==2.9.0.post0
    # via -r requirements.in
six==1.16.0
    # via python-dateutil
tomli==2.0.1
    # via
    #   coverage
    #   pytest
zipp==3.19.2
    # via importlib-metadata
